src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -30,12 +30,19 @@ internal class RequestTracingConstants
         public const string KeyVaultConfiguredTag = "UsesKeyVault";
         public const string KeyVaultRefreshConfiguredTag = "RefreshesKeyVault";
         public const string ReplicaCountKey = "ReplicaCount";
+        public const string FeatureFlagUsesTelemetryTag = "Telemetry";
+        public const string FeatureFlagUsesSeedTag = "Seed";
+        public const string FeatureFlagMaxVariantsKey = "MaxVariants";
+        public const string FeatureFlagUsesVariantConfigurationReferenceTag = "ConfigRef";
```

Comment:
Do we want to track inline config as well?

Reply:
Overall, it seemed like configuration reference was the important one for us to track since it's more of a feature while inline config is kind of the standard usage, but I could add it if it's helpful for us to track people who have no configuration on their variant at all versus inline config.

Reply:
It doesn't feel necessary.

Reply:
I agree, doesn't seem necessary.

Reply:
Ok sounds good